packages/build/webpack/constants.js

```diff
@@ -23,21 +28,14 @@ if (process.env.LINK) {
     }
 
     labkeyUIComponentsPath = process.env.LABKEY_UI_COMPONENTS_HOME + '/packages/components/src';
+    // lastIndexOf just in case someone is weird and has their LKS deployment under a directory named modules.
+    const lkModulesPath = cwd.slice(0, cwd.lastIndexOf('modules') + 1);
```

Comment:
We currently have this directory structure for some of our multi-module repos `server/modules/xModules/` (e.g., `server/modules/customModules` and `server/modules/ehrModules`).  While this will still do the right thing since it's case-sensitive, there's potential that the name of a module container repo has the word `modules` in it, so maybe look for `/modules/` (though the slashes will have to be platform-specific, of course).

Reply:
This is a search against the entire directory name, so if it has the word `modules` in it it will fail, unless the entire word is modules. I could change this to just find the first one, and anyone who is weird would have to stop being weird. Not sure which is more likely though.

Reply:
In this case `cwd` is an array of directory names.

Reply:
Ah, didn't realize it's an array. In that case, never mind. :-)